### PR TITLE
Update airbnb.swiftformat

### PR DESCRIPTION
**Pull request by SofiaMNC**
#### Summary

Add the following rules:
- [wrapMultilineStatementBraces](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapmultilinestatementbraces)
- [wrapConditionalBodies](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapconditionalbodies)
- [wrapEnumCases](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapenumcases)
- [wrapSwitchCases](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapswitchcases)

#### Reasoning
- Without the `wrapMultilineStatementBraces` rule, I feel it makes it hard to see where the conditions / function parameters end and where the action starts. Also, this rule is formulated in AirBnB's style guide, but it seems to be missing from their `.swiftformat` file.
See this example, where it is hard to see where the `if` conditions end and where the action starts:
    - with the current setting
Capture d’écran 2022-04-14 à 09 57 11
    - with the `--wrapMultilineStatementBraces` rule
Capture d’écran 2022-04-14 à 09 57 25

- The other 3 rules (`wrapConditionalBodies`, `wrapEnumCases`, `wrapSwitchCases`) enforce what we already have in our guidelines (see the examples in the doc linked above for each one of them), so I feel it would be a good idea to include them so we don't have to think about it.

_Please react with 👍/👎 if you agree or disagree with this proposal._

### CODE REVIEW

**Review by waroo — CHANGES_REQUESTED** [review]
For each new rule proposed, there should be an explanation and example like Airbnb does:

image

**Review by waroo — COMMENTED** [review]
I don't see wrapEnumCases and wrapSwitchCases currently in our guidelines so these are brand new rules.

**Comment by SofiaMNC** [comment]
> I don't see wrapEnumCases and wrapSwitchCases currently in our guidelines so these are brand new rules.

I thought there were PRs where that came up. I guess I was mistaken, I can remove them.

**Comment by waroo** [comment]
If everyone agrees to these two new rules then that is ok to keep, I just
want to make sure it's what people want to change to since it is something
new.

**Review by joaopdcgarcia — APPROVED** [review]

**Review by waroo — CHANGES_REQUESTED** [review]
Each rule should have one to two lines explaining what the rules is and does in plain English.  Justification for why the rule is beneficial is also nice.  Note how the first part is a brief explanation of the rule and is bolded and the second part is more of a description or clarification or justification.

Also the -- rule rulename text is unnessary as it is included int the [SwiftFormat][rulename] block.

See all the rules at https://github.com/slumberGroup/swift for different examples.

**Comment by waroo** [comment]
Don't forget to open a PR or issue to the upstream branch at https://github.com/airbnb/swift/pulls?q=is%3Apr+is%3Aclosed for the wrapMultilineStatementBraces rule

File: resources/airbnb.swiftformat [hunk 1]
```diff
@@ -59,6 +59,10 @@
 --rules wrap
 --rules wrapArguments
 --rules wrapAttributes
+--rules wrapConditionalBodies
```
Comment (standinga) [inline resources/airbnb.swiftformat:62]: I'm confused because this is what I'm getting with the current Slumber rules. Do you have the latest SwiftFormat as mentioned in the file (0.50-beta-1)
Screenshot 2022-04-21 at 13 55 14
Btw. I personally find it harder to read when the braces are on new lines, also it's a bit confusing that for non-multiline conditions they are on new lines. But I can live with it.
Anyway ,please check if you have the latest SwiftFormat required by the rules we are currently using as for me they seem to be already infered without the need to explicitly putting them here.

**Review by standinga — CHANGES_REQUESTED** [review]

File: resources/airbnb.swiftformat (hunk 1 above)
Comment (madimedia) [reply resources/airbnb.swiftformat:62]: Yes, I have checked, and the `wrapMultilineStatementBraces` rule is definitely missing from Airbnb's swift format file.

Isn't the example you're showing old code? From when we used to have this rule? I just checked out BTF and tried adding new, unformatted code:
Capture d’écran 2022-04-21 à 09 37 16
And after running Swift Format:
Capture d’écran 2022-04-21 à 09 36 52

The brace is not moved to the line below as you can see, so it seems to have the issue I was mentioning, which would be normal since the rule is not in the .swiftformat file.

Comment (SofiaMNC) [reply resources/airbnb.swiftformat:62]: Good catch @standinga! I have the latest stable version. 
In the beta, `wrapMultilineStatement` is removed as a rule and becomes the parameter `--multiline-stmt` to `--rules braces`, with the behavior we want as a default, which is why it works:
Capture d’écran 2022-04-21 à 10 21 28

However, I notice it is not the "real" SwiftFormat, but a fork by AirBnb off the official Swift Format. Are we ok to use this instead @waroo? If so,  I'll remove the rule from this PR.

**Review by standinga — APPROVED** [review]

**Review by waroo — APPROVED** [review]
